libs/design/src/atoms/form/checkbox-set/checkbox-set.component.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { Component, DebugElement } from '@angular/core';
+import { FormGroup, FormControl, ReactiveFormsModule } from '@angular/forms';
+import { By } from '@angular/platform-browser';
+
+import { DaffCheckboxSetComponent } from './checkbox-set.component';
+import { DaffCheckboxModule } from '../checkbox/checkbox.module';
+
+@Component({
+  template: `
+ 
+<daff-checkbox-set [formGroup]="checkboxGroup" name="toppings">
+  <daff-checkbox formControlName="lettuce" value="lettuce">Apple</daff-checkbox>
+  <daff-checkbox formControlName="tomato" value="tomato">Grape</daff-checkbox>
+  <daff-checkbox formControlName="pickle" value="pickle">Peach</daff-checkbox>
+</daff-checkbox-set>
+  `
+})
+class CheckboxEmbeddedComponent {
+  checkboxGroup = new FormGroup({
+    lettuce: new FormControl(),
+    tomato: new FormControl(),
+    pickle: new FormControl()
+  });
+}
+
+describe('DaffCheckboxSetComponent', () => {
```

Comment:
Why does this spec contain the `daff-checkbox` if it doesn't test any behavior with the `daff-checkbox`?